deploy/openshift/sws.yaml

```diff
@@ -45,22 +50,41 @@ objects:
     to:
       kind: Service
       name: sws
+- apiVersion: v1
+  kind: ConfigMap
```

Comment:
Do we not want the configmap to be separate from the template?
Normally someone will be making changes to the configmap for their purposes (username/password, using another port, etc) and they might want to redeploy the deployment and leave the configmap alone

Reply:
Granted, I am not sure how things are typically done, but the port number as an example can be overridden via the env var SERVICE_PORT (which the makefile passes on the command line when the template is processed). Thus I thought it just easier to roll the config map into the one yaml rather than have two yaml files.

But I could split it back out again and use a template for the config map (we need it as a template so we can pass the parameter for the port thus allowing someone to change the port by simply setting the env var in the make openshift-deploy command or the oc process command via -p).